Cookbook/Current.md

```diff
@@ -0,0 +1,18 @@
+# How to use `Current` in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+It's purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### do
+- access `Current` **only** from `Builder`s layer;
+- pass the required dependencies down the chain;
+- contribute to it;
```

Comment:
How? What can we add to it and what should we not?

Reply:
I think it doesn't make sense.. once all `appDependencies` are moved into Current, no need to contribute to it. As Giorgos said, it's a matter to follow the pattern os other properties: with default value or set asynchronously.

I changed to the mutation when it's allowed. 😉